torchvision/prototype/datasets/_builtin/celeba.py

```diff
@@ -0,0 +1,204 @@
+import csv
+import io
+import pathlib
+from typing import Any, Callable, Dict, List, Optional, Tuple, Sequence
+
+import torch
+from torch.utils.data import IterDataPipe
+from torch.utils.data.datapipes.iter import (
+    Mapper,
+    Shuffler,
+    Filter,
+    ZipArchiveReader,
+)
+from torchdata.datapipes.iter import KeyZipper
+from torchvision.prototype.datasets.utils import (
+    Dataset,
+    DatasetConfig,
+    DatasetInfo,
+    GDriveResource,
+    OnlineResource,
+)
+from torchvision.prototype.datasets.utils._internal import INFINITE_BUFFER_SIZE
+
+
+class CelebACSVParser(IterDataPipe):
+    def __init__(
+        self,
+        datapipe,
+        *,
+        has_header,
+        delimiter=" ",
+        skipinitialspace=True,
```

Comment:
I understand this is originally from the `csv` module, but this is inconsistent with the rest of our parameter naming convention. So unless we have a strong reason to keep this, I would suggest to change to `skip_initial_space`

Same for `other_fmt_params` below